airflow_home/dags/modules/citytest_sf_appointments.py

```diff
@@ -10,7 +10,7 @@
 class CityTestSFAppointments(Core):
     """Functions related to business logic around CityTestSF appts."""
     @staticmethod
-    def parse_appointment(appointment):
+    def parse_appointment(appointment, extra_fields=None):
```

Comment:
doesn't seem like you're using this?

Reply:
It is used at https://github.com/SFDigitalServices/airflow-service-py/blob/d3c713464ce7384dd30f2911b6bb948edd5a6efa/airflow_home/dags/scripts/citytest_sf_appointments_report.py#L44-L46. I do feel its a pretty fragile logic tho.